### PR TITLE
Move telemetry to receiver tab

### PR DESCRIPTION
**Pull request by haslinghuis**
Move telemetry to receiver tab.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:1220]: Moving them here, I suppose maybe there is other place where we need to remove it? There are translations in this other place?
As I always recommend with new translations, please add a `description` with information to the translators. In this case I think is difficult to be wrong, but it always help.

**Review by McGiverGim — APPROVED** [review]
Approved. An screen capture always help with this kind of PR ;)

**Comment by haslinghuis** [comment]
Actually not happy with layout but spent hours to get it right without any success.

image

**Comment by McGiverGim** [comment]
Yes, at some moment we need to remove all of this features stuff and convert them in standard msp properties.
But for the moment, I think it's ok.

**Review by blckmn — APPROVED** [review]

**Pull request merged**
